<launch>
  <arg name="mav_name" default="$(optenv MAV_NAME)"/>

  <group ns="$(arg mav_name)">

    <!--<node pkg="snavquad_interface" type="vislam_point_overlay.py" name="vislam_point_overlay" output="screen"/> -->
    <node pkg="snavquad_interface" type="point_overlay" name="vislam_point_overlay" output="screen">
      <remap from="image_raw" to="dfc/image_raw"/>
      <remap from="image_overlay" to="dfc/image_overlay"/>
    </node>

    <node pkg="kr_qualcomm_interface" type="snav_publisher" name="snav_publisher" output="screen">
      <param name="rpm_rate" value="100" />
      <param name="attitude_rate" value="100" />
      <param name="status_rate" value="5.0" />
    </node>

  </group>
</launch>